<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>照相机made by郜创新</string>
  </property>
  <property name="windowIcon">
   <iconset resource="heihei.qrc">
    <normaloff>:/Camera2.png</normaloff>:/Camera2.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>-10</y>
      <width>621</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ImageView"/>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>-10</y>
      <width>91</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="heihei.qrc">
      <normaloff>:/back1.png</normaloff>:/back1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>800</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCapture">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="heihei.qrc">
      <normaloff>:/Camera2.png</normaloff>:/Camera2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSave">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>540</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="heihei.qrc">
      <normaloff>:/load.png</normaloff>:/load.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonQuit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>-10</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="heihei.qrc">
      <normaloff>:/Close.png</normaloff>:/Close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ImageCapture">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>454460</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="photocut">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>截图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shoot">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>连拍</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopShoot">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>停止连拍</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grayButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>黑白</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>330</y>
      <width>881</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="redButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>480</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>暖色</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>610</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reduce0Button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>520</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>失真</string>
    </property>
   </widget>
   <widget class="QPushButton" name="originalButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>570</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>还原</string>
    </property>
   </widget>
   <zorder>pushButton</zorder>
   <zorder>horizontalWidget</zorder>
   <zorder>buttonCapture</zorder>
   <zorder>buttonSave</zorder>
   <zorder>buttonQuit</zorder>
   <zorder>ImageCapture</zorder>
   <zorder>label</zorder>
   <zorder>photocut</zorder>
   <zorder>shoot</zorder>
   <zorder>stopShoot</zorder>
   <zorder>grayButton</zorder>
   <zorder>line</zorder>
   <zorder>redButton</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>reduce0Button</zorder>
   <zorder>originalButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>模式</string>
    </property>
    <addaction name="action555"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action555">
   <property name="text">
    <string>555</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="heihei.qrc"/>
 </resources>
 <connections/>
</ui>
